Code including two modals for Wookie Welcome page plus added empty li…

DrKalen opened the pull request:
…ne at end of rubocop.yml and booyah box and background for pages

Jorge-FHP commented:
Looks good to me!

DrKalen commented:
Great! Thanks for checking it.

Best,
Kalen


Dr. Kalen Hammann ("Dr. K")
Communication Specialist
Ofc:727-733-2178
Cell: 1-303-589-7107
See clips at www.BookDrK.com <http://www.bookdrk.com/>

On Wed, May 17, 2017 at 5:58 PM, Jorge-FHP <notifications@github.com> wrote:

> Looks good to me!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/team-LetTheWookieWin/chess-app/pull/18#issuecomment-302243447>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AX7YP24wxr8kcYxcP3WmsCxuGWwvsLCgks5r622ggaJpZM4NeFTM>
> .
>

DrKalen commented:
Your chessboard looks like it's probably great, but I don't fully
understand the code. Nice job of implementing a table!

Best,
Kalen



Dr. Kalen Hammann ("Dr. K")
Communication Specialist
Ofc:727-733-2178
Cell: 1-303-589-7107
See clips at www.BookDrK.com <http://www.bookdrk.com/>

On Wed, May 17, 2017 at 6:03 PM, Kalen Hammann <kalen7@gmail.com> wrote:

> Great! Thanks for checking it.
>
> Best,
> Kalen
>
>
> Dr. Kalen Hammann ("Dr. K")
> Communication Specialist
> Ofc:727-733-2178
> Cell: 1-303-589-7107
> See clips at www.BookDrK.com <http://www.bookdrk.com/>
>
> On Wed, May 17, 2017 at 5:58 PM, Jorge-FHP <notifications@github.com>
> wrote:
>
>> Looks good to me!
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/team-LetTheWookieWin/chess-app/pull/18#issuecomment-302243447>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe-auth/AX7YP24wxr8kcYxcP3WmsCxuGWwvsLCgks5r622ggaJpZM4NeFTM>
>> .
>>
>
>

Jorge-FHP commented:
This is how it looks like, it's plain but I ran into some problems and I didnt have much time to work on it much.

<img width="917" alt="screen shot 2017-05-17 at 5 06 59 pm" src="https://cloud.githubusercontent.com/assets/25312157/26178374/549465d8-3b23-11e7-8152-cb1004d44a4a.png">

jacobwgillespie commented on app/views/static_pages/index.html.erb:
```diff
@@ -1,2 +1,109 @@
-<h1>Home Page</h1>
+<html>
+  <body>
+    <br />
```
So, `<br />` tags are used for line breaks inside a paragraph, or when you need to move to the next line without starting a new paragraph.  They are not necessarily supposed to be used for formatting, as in you'd use them when the paragraph needed a line break, not when you want blank space.  For blank space, there's the css `margin` and `padding`.

Anyways, it's probably okay to leave them here for now, unless you want to remove them, but by the end we should move all the spacing over to CSS rules.

DrKalen replied:
I'll leave them here for now- not sure how I would do CSS spacing around a booyah-box. Do I add another part to the class?

jacobwgillespie commented on app/views/static_pages/index.html.erb:
```diff
@@ -1,2 +1,109 @@
-<h1>Home Page</h1>
+<html>
```
`<html>` and `<body>` tags should only live inside layouts (like `application.html.erb`) as the contents of the views will get embedded inside the layout file (it replaces the `<%= yield %>` tag), and there can only be one `<html>` or `<body>` tag.  You should just be able to remove them from here and everything should still work.  👍

DrKalen replied:
OK, I'll take them out.

DrKalen commented on rubocop.yml:
```diff
@@ -0,0 +1,24 @@
+AllCops:
```
I'm confused. I thought we had to have the rubocop.yml file and that's where I got rid of the Style/ClassAndModuleChildren that was giving Tyna trouble. I just included it in this commit because I had added a final empty line to the file.

jacobwgillespie replied:
This file is the `rubocop.yml` file _without_ the `.` at the beginning, so Rubocop isn't actually reading it.  The `.rubocop.yml` file is the real one, and it looks like it already has `Style/ClassAndModuleChildren` disabled in it.

DrKalen replied:
Got it. Thanks.

DrKalen commented on app/views/static_pages/index.html.erb:
```diff
@@ -1,2 +1,109 @@
+        4.  If your partner is offline, click the “Send an email" button on the game board. (If at any point you are online but your partner is not, you can click the “Send an email” button to notify your partner when it’s his/her move.  )
+        </p>
+        <p>
+        5.  If you are offline and receive an email, log in and make your next move. (If you are offline when your partner makes a move, your partner can click the “Send an email” button to notify you that it’s your turn.)
```
Good eye! Thanks.

DrKalen commented:
I just made a commit with HTML changes. Do I now ask for another pull request, or do this changes become part of the same one?

DrKalen commented:
Aah - I see, the pull request automatically includes them. Yay GitHub! :-)

DrKalen commented:
Just added a commit with these changes:
Changed navbar link from “Sign in “ to “log In”
Added top margin to booyah-box class
Added booyah boxes  and internal spacing to Log In and Sign Up forms

jacobwgillespie commented:
Looking good!  Looks like `gem 'simple_form'` is missing from the `Gemfile` in this PR, which is causing Travis CI to fail the build.  And there's a conflict in `master.scss`, but once those two things are fixed this should be good to merge.

DrKalen commented:
I've resolved the conflicts on .rubocop.yml and master.scss (in each case by adding what was in BOTH files) and added 'simple_form' to Gemfile. All ready to merge!
